packages/x-components/src/x-modules/facets-next/store/types.ts

```diff
@@ -54,6 +50,18 @@ export interface FacetsNextMutations {
    * update.
    */
   setFacetGroup(facetGroupEntry: FacetGroupEntry): void;
+  /**
+   * Adds the filter to the {@link FacetsNextState.filters | filters} record.
+   *
+   * @param filter - The filter to add.
+   */
+  setFilter(filter: Filter): void;
+  /**
+   * Sets the {@link FacetsState.query} property.
```

Comment:
`   * Sets the {@link FacetsNextState.query} property.`

Reply:
Buen ojo you too